guides/common/modules/con_content-sources-in-project.adoc

```diff
@@ -0,0 +1,14 @@
+[id="content-sources-in-{ProjectNameID}_{context}"]
+= Content sources in {Project}
+
+The Red{nbsp}Hat subscription manifest determines what Red{nbsp}Hat repositories are accessible from your {ProjectServer}.
+Once you enable a Red{nbsp}Hat repository, an associated {Project} product is created automatically.
+For distributing content from custom sources you need to create products and repositories manually.
+Red{nbsp}Hat repositories are signed with GPG keys by default, and it is recommended to create GPG keys also for your custom repositories.
+
+Yum repositories that contain only RPM packages support the *On demand* download policy, which reduces synchronization time and storage space.
```

Comment:
"contain only RPM packages" < does this exclude errata? I think this statement is too narrow.

Alternative: rephrase to make it more generic such as "Different content types support different download policies. For more information, see https://docs.theforeman.org/nightly/Managing_Content/index-katello.html#Download_Policies_Overview_content-management"

Reply:
"Repositories of content type _yum_" would be more technically accurate (but not as readable)..